File: QTD2.Infrastructure/Model/ILA_CBT_ScormUpload/ScormUploadAddOptions.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using QTD2.Infrastructure.Model.Shared;

namespace QTD2.Infrastructure.Model.ILA_Cbt_ScormUpload
{
    public class ScormUploadAddOptions
    {
        public int CbtId { get; set; }        

        public FileUploadModel File { get; set; }
    }
}

File: QTD.SPA/src/app/_DtoModels/Scorm/ScormUploadAddOptions.ts
import { FileUploadModel } from "./FileUploadModel";

export class ScormUploadAddOptions{
    cbtId:number;
    file: FileUploadModel;

    constructor(cbtId:number,file:FileUploadModel){
        this.cbtId=cbtId;
        this.file=file;
    }
}
